runtime/devnet/src/extensions.rs

```diff
@@ -78,6 +73,17 @@ where
 	}
 }
 
+pub(crate) fn convert_to_status_code(error: DispatchError) -> u32 {
+	match error {
+		_ => {
+			let mut encoded_error = error.encode();
+			// Resize the encoded value to 4 bytes in order to decode the value in a u32 (4 bytes).
+			encoded_error.resize(4, 0);
```

Comment:
Would encoded_error[0..4] not do the same?

Reply:
Taken from [substrate](https://github.com/paritytech/polkadot-sdk/blob/0b11c27e3eedbe53d646c6fd2ec45e9e898b00bd/substrate/frame/support/procedural/src/pallet/expand/error.rs#L165)